assets/sass/v2/_webauthn.scss

```diff
@@ -21,5 +21,13 @@
     .idx-webauthn-enroll-text {
       margin-bottom: 20px;
     }
+
+    .setup-webauthn-residentkey-text {
+      margin-bottom: 25px;
+      .setup-webauthn-residentkey-link {
+        color: $primary-button-bg-color;
+        padding-left: 5px;
```

Comment:
👋 Any reason this link is getting a special (background?) color and padding?

Exceptions can make it more difficult for customers to theme and customize the widget.

Reply:
good catch, padding is not necessary, color is needed but changed to the common `link-text-color`, thanks a lot